app/editor/src/features/content/list-view/utils/makeFilter.ts

```diff
@@ -37,6 +44,59 @@ export const makeFilter = (
   return result;
 };
 
+/**
+ * Creates a IContentFilter that can be passed to the API hook endpoint.
+ * Cleans up the data to ensure it matches what is expected by the API.
+ * @param filter Filter object
+ * @returns new IContentFilter object.
+ */
+export const getFilter = (
```

Comment:
You renamed this to getFilter but the file name is makeFilter.  Why change the name of the function?

Reply:
This is a new function instead of renaming the original function makeFilter, as currently the original function makeFilter is still being used by other parts of the system, when we fully refactor to use the new Elasticsearch endpoint, we might need to clean up and remove the original function.

Reply:
Then create a new file for the function.  It's a pain to find functions in javascript unless they exist in their own file.  Follow the convention.

Reply:
Updated now, initially thought that when the refactoring is done in a following ticket, we can just remove the old function and rename the file to the new one.

Reply:
Ah I see.  Let's stick with conventions for readability and maintainability.  When the other function is no longer needed, we'll delete that file.